specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/virtualWan.json

```diff
@@ -3122,7 +3122,7 @@
           "200": {
             "description": "Request successful. Returns the details of all the VirtualHubRouteTableV2s for the VirtualHub.",
             "schema": {
-              "$ref": "#/definitions/ListHubVirtualNetworkConnectionsResult"
+              "$ref": "#/definitions/ListVirtualHubRouteTableV2sResult"
```

Comment:
As you change the schema of the result, will this be a breaking change for this API?

Reply:
Yes, i just added the api yesterday and this did not catch my attention, so this commit is fixing the error.